apps/api/src/admin/participant_manager.py

```diff
@@ -18,10 +19,21 @@ class Participant(UserRecord):
 
 
 async def get_attending_applicants() -> list[Participant]:
-    """Fetch all applicants who have a status of ATTENDING"""
+    """Fetch all applicants who have a status of ATTENIDNG, WAIVER_SIGNED, CONFIRMED,
+    or WAITLISTED."""
```

Comment:
Thought: unsure if function name `get_attending_applicants` is still accurate.
Chore: typo on `ATTENIDNG`